tests/regression/test_oracle_report_with_notifier.py

```diff
@@ -0,0 +1,182 @@
+import pytest
```

Comment:
This test makes no sense for mainnet yet, thus must be expluded some how from the tests batch run